assets/dev/js/editor/elements/views/base.js

```diff
@@ -1085,7 +1085,7 @@ BaseElementView = BaseContainer.extend( {
 		return helper;
 	},
 
-	getDraggableElement() {
+	getElement() {
```

Comment:
Maybe `getDOMElement` because `Element` is also Elementor element and we have function that called `getElementType`